docs/docs/running-hello-world.md

```diff
@@ -1,55 +1,56 @@
-# Compiling and Running your First Enclave
+# Compiling and running your first enclave
 
 ## Introduction
 
-This tutorial describes how to compile and run the sample hello world app, which you can find in the 
-[Conclave tutorials repository](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world). This app contains the `ReverseEnclave`, 
-which reverses a string provided by the client and passes it back.
+This tutorial describes how to compile and run the sample hello world app, which you can find in the
+[Conclave tutorials repository](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world).
+This app contains the `ReverseEnclave` function, which reverses a string provided by the client and passes it back.
 
-We are first going to run the app in [mock mode](enclave-modes.md),
-which is the simplest (and least realistic) of the enclave modes.
-We will explore running in other modes [later on](#beyond-mock-mode).
+Follow the instructions to run the app in [mock mode](enclave-modes.md). After running it in mock mode, you
+can proceed to [run the app in other modes](#beyond-mock-mode).
 
-If you get stuck, join our [discord server](https://discord.com/invite/dDaBGqGPre) and ask for help!
+If you get stuck at any step, please [talk to us on Discord](https://discord.gg/zpHKkMZ8Sw).
 
 ## Prerequisites
 
-* You need JDK 17 installed in order to build and run the app.
 * This tutorial assumes you've read and understood the [conceptual overview](enclaves.md)
   and [architecture overview](architecture.md).
+* You need JDK 17 or higher installed. You can download it [here](https://www.oracle.com/java/technologies/downloads/).
 
 ## Compile the sample application
 
-Clone and navigate to the hello world sample.
+Like all Conclave apps, this sample app has an [enclave and a client](architecture.md#primary-entities). The enclave
+runs inside a host, which is also a part of the sample app.
+
+To get the sample app and compile it:
+
+1. Open a command line interface.
```

Comment:
I don't think we need to tell users to open a command line interface. We don't mention it anywhere else in our docs before the bash commands. I would just remove this line.

Reply:
I understand your angle. 

However, not every developer who reads this tutorial will know they must open two command lines - one for the host and one for the client. That's why I have added this step only in the tutorial. In most other cases, we don't want to tell this explicitly. But here, it will help a section of the readers.